packages/code-connect/components/Slider/Slider.figma.tsx

```diff
@@ -0,0 +1,48 @@
+import figma from '@figma/code-connect';
+import { Button, Slider } from '@patternfly/react-core';
+import MinusIcon from '@patternfly/react-icons/dist/esm/icons/minus-icon';
+import PlusIcon from '@patternfly/react-icons/dist/esm/icons/plus-icon';
+
+// TODO: FIGMA: Define left/right actions
+// Documentation for Slider can be found at https://www.patternfly.org/components/slider
+
+figma.connect(
+  Slider,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=3155-97956&t=IzSunfrnw18ti37Y-11',
+  {
+    props: {
+      // boolean
+      isInputVisible: figma.boolean('Value input'),
+      minmaxValues: figma.boolean('Min/max values'),
+      valueInput: figma.boolean('Value input'),
```

Comment:
Rename here to match prop name `isInputVisible` (used correctly in example code below) 
```suggestion
      isInputVisible: figma.boolean('Value input'),
```